python/ribasim/ribasim/model.py

```diff
@@ -33,7 +33,7 @@
 class Solver(BaseModel):
     algorithm: Optional[str]
     autodiff: Optional[bool]
-    saveat: Optional[List[float]]
+    saveat: Optional[Union[float, List[float]]]
```

Comment:
```suggestion
    saveat: float | List[float] | None
```

Yes, this requires Python 3.10 but this is just so much more readable

Reply:
I'm fine with requiring 3.10, but shouldn't the pyproject.toml and other things then also be updated to reflect that?

Reply:
Good point, let's look into that for a different PR

Reply:
Does that mean you approve this PR? ;)